.circleci/config.yml

```diff
@@ -291,6 +295,11 @@ defaults: &defaults
       only: *tag_regex
   context:
     - build
+    - slack
+  post-steps:
+    - slack/notify:
+        event: fail
+        branch_pattern: "master,chore/slack-notification"
```

Comment:
does `chore/slack-notification` need to stay?

Reply:
Good catch!